src/org.freedesktop.PackageKit.xml

```diff
@@ -476,6 +476,17 @@
       </doc:doc>
     </signal>
 
+    <!--*********************************************************************-->
+    <signal name="InstalledChanged">
```

Comment:
This information is usually not known, similarly as in the `UpdatesChanged` signal and the other one for the repo list change. In case of the dnf backend, the signal is emitted when the underlying RPM db changes, which is recognized by a GFileMonitor. The comment says it, "maybe something in the list of installed apps changed", no guarantee there was really anything installed or uninstalled (or reinstalled).